### DISCUSSION TITLE
【今日观点】死水还阴啊？

### DISCUSSION THREAD

**Discussion by wangzihhy** [share121/inter-knot#2706] [其他]
![29baa356cfb137c680615c6f5770065](https://github.com/user-attachments/assets/3b3ca038-dec0-4042-b8ef-531657f8efd7)
我认为死水是一个极其需要尊重的老角色，因为他在火影忍者手游存在的这八年内，给这个游戏带来了巨大的热度，如果没有死水，火影忍者手游可能就没有今天，所以死水应该是火影手游全体尊重的对象，死水不但制作精良，而且特别需要操作的手法，如果段位没有3000分玩这个忍者就不能发挥他的实力，一但不能完全发挥实力，这个忍者就是无比鸡肋的忍者，此时需要的就是意识和手法来弥补对这个角色的操作不足，当技术大成时即可打出号称游戏最需要操作的玩法，人们都称为z字闪。这个忍者真的没有你们想的那么简单，不仅仅是在游戏中，在动漫中死水也是一个沉稳伟大的父亲，虽然没给鸣人一个完整的童年，不过他在战场上，在村子中，都是耀眼的领袖，就和游戏中需要操作的阳间战神一样，黄色闪光，你抓不住

**Comment by nidaye-png** [comment]
叮，抱歉您的句子当中可能含有『水』，我可以认为您对死水进行了评价。
你刚刚是评价了死水是吗？我个人觉得波风水门【秽土转生】真的是最需要操作的忍者了。我对操作的定义就是这件事你知道怎么做但是你做不到。在目前的所有忍者中只有死水能让我有这种体验。不说别的，被先手了替身2A插标闪走，这套操作没玩个几百把是真的做不到的，90%的人想做到这套操作都会发现自己2A根本插不出标来，训练营和决斗场的心理压力是不一样的，你在训练营就算能百分百做到2A插标，决斗场也90%的情况是2A插不出来。我玩骨架白面具破面小土豆，看到对面玩死水，就知道自己低人一等了，对面换位金身博弈我也是心服口服的，抓不到对面只能说我玩不明白死水，死水玩家摸头发表情我也是羞愧难当的，毕竟玩死水的绝对是全火影最需要操作的，我这种玩脑忍者的不能去攻击操作忍者的。我以后在火影忍者手游里，我以后再也不敢说秽土水门这个忍者的任何坏话，我承认死水这个忍者确实是要技术的，cd久的，阳间的，老忍者的，没抓取的，没反手的，被起手玩不了的，没有通灵秘卷t3的，伤害低的，要博弈的，z闪是难触发的，非常极端的，会被骨架克制的，打不过儿子黄猿的

**Comment by LuAn-afl** [comment]
你评价了死水对吧？
我个人觉得波风水门【秽土转生】真的是最需要操作的忍者了。我对操作的定义就是这件事你知道怎么做但是你做不到。在目前的所有忍者中只有死水能让我有这种体验。不说别的，被先手了替身2A插标闪走，这套操作没玩个几百把是真的做不到的，90%的人想做到这套操作都会发现自己2A根本插不出标来，训练营和决斗场的心理压力是不一样的，你在训练营就算能百分百做到2A插标，决斗场也90%的情况是2A插不出来。我玩骨架白面具破面小土豆，看到对面玩死水，就知道自己低人一等了，对面换位金身博弈我也是心服口服[头盔]

**Comment by SOGOsogo** [comment]
死水不阴握持
